quinn-proto/src/connection/mod.rs

```diff
@@ -2074,7 +2095,11 @@ where
                 if self.total_authed_packets > 1 {
                     return Ok(());
                 }
-                if packet.payload.chunks(4).any(is_supported_version) {
+                let supported = packet.payload.chunks(4).any(|x| {
+                    self.supported_versions
+                        .contains(&u32::from_be_bytes(x.try_into().unwrap()))
+                });
+                if supported {
```

Comment:
I can add back the `is_supported_version` utility.

Reply:
Nah, this is fine.